frontend/src/components/Course/Course.tsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 
 import { shortifyCourseDescription } from 'utils/helpers/shortifyCourseDescription';
+import convertStatusToProgress from 'utils/helpers/convertStatusToProgress';
```

Comment:
just convertStatus. in the previous context, such a naming would be correct. But after rebase, it should be just convertStatus

Reply:
i took already in use function and removed ConvertStatus because it was repetitive code as in convertStatusToProgress